### ISSUE TITLE
Invalidate/NotifyDataSetChanged Does Not work as expected

### ISSUE THREAD

**Issue by Koalified** [alamkanak/Android-Week-View#211]
My Issue is as Follows:

I have tried everything to trigger a dynamic update short of doing a full activity.recreate which would be less than preferable when more data is put in the calendar and converted to WeekViewEvents.

So from what I gather, the interface onMonthChange populates the weekview events from either a custom source or a one time pointer to a List of weekview events. I have tried both options but because the prefetched interface populates 3 months worth of events, it is almost as though the interface has a 3 part recursive method for three different months which according to my log is true:
 Eval: M/Y10/2015 start M/Y: 10/2015
 Eval: M/Y10/2015 start M/Y: 10/2015
 Week View Events: 2
 Eval: M/Y11/2015 start M/Y: 10/2015
Eval: M/Y11/2015 start M/Y: 10/2015
Week View Events: 0
 Eval: M/Y12/2015 start M/Y: 10/2015
Eval: M/Y12/2015 start M/Y: 10/2015
Week View Events: 0

As just the output where the first set of values (M/Y) is the evaluated values and the second set are the comparables. The only margin of success I have had thus far is to not check whether the month fits the evaluated month and add/edit/delete from an encapsulated List relative to the calling class.

I guess the TL;DR version of this would be is there some way to force the view to either do a complete refresh, only evaluate one month, or another means of getting the desired behavior of not having to completely reload the activity to have new weekviewevents appear on the schedule calendar?

**Comment by Koalified** [comment]
Finally Found one solution that works:
If I repopulate my external schedules then call getMonthChangeListener and call onMonthChange to the schedules start datetime and THEN call notifydatasetchanged(); it works as expected. So I guess alls well that ends well. 

I just posted because I had been stuck on that issue for a matter of 3/4 days. Kinda wish there was a more intuitive overriden way to recall onMonthChange to redraw everything but at least I have something in the mean time.

Thanks!
